lib/FundDistribution/FundDistributionFields/FundDistributionFieldsFinal.js

```diff
@@ -174,6 +174,7 @@ const FundDistributionFieldsFinal = ({
             onChange={selectFund}
             validate={required ? validateRequired : undefined}
             validateFields={[name]}
+            onBlur={() => debouncedValidate(fields.value)}
```

Comment:
I'm a bit confused about this line. Is it really required here? I believe final form validates fields either on change or on blur automatically based on form config,

Reply:
@Yuliia-Dovhal Yes, this is the workaround to display the required message for ExpanseClassId. When I removed it, the error message isn't displayed. If you remember there was a case the error message was displayed when the selection has been touched.

https://github.com/folio-org/stripes-acq-components/assets/116072773/75ccb2b9-a304-4ac2-80b9-1654750e3dd5